kalite/distributed/views.py

```diff
@@ -212,6 +213,21 @@ def search(request):
         'category': category,
     }
 
+def create_superuser(request):
```

Comment:
do you mean the validation part when you talk about automation? I will refactor this with Django forms which handles the validation for us.